### ISSUE TITLE
`fat create` tries to open a file named "buffered:<...>"

### ISSUE THREAD

**Issue by timbertson** [mirage/ocaml-fat#65]
When creating a fat file it tries to literally open `buffered:<...>` when it comes time to connect:

```
$ strace -f fat create out.fat 5MiB
( ... )
[pid 10743] open("out.fat", O_RDWR|O_CREAT, 0644) = 4
( ... )
[pid 10743] write(4, "All work and no play makes Dave "..., 512) = 512
( ... )
[pid 10742] open("buffered:out.fat", O_RDWR) = -1 ENOENT (No such file or directory)
[pid 10742] open("buffered:out.fat", O_RDONLY) = -1 ENOENT (No such file or directory)
[pid 10742] write(2, "fat: connect buffered:out.fat: f"..., 51fat: connect buffered:out.fat: failed to open file
) = 51
```

Adding `--unbuffered` is a simple workaround, but it's a bit odd.

I feel like this could well be a dependency version issue, as the code looks like it should handle this. I'm building under opam2nix, which should produce the same solve results as opam does (unless there's bugs, which there definitely could be), so here are all my dependency versions in case that's relevant:

https://gist.github.com/timbertson/0598e79a43a0483a033ad5a0447942a6

**Comment by yomimono** [comment]
I can reproduce this with commit b0880ff of `ocaml-fat` and `mirage-block-unix` version 2.9.0 , the latest as of this writing.

**Comment by philipderrin** [comment]
This was fixed by #75, but there hasn't been a release since that was merged.

**Comment by djs55** [comment]
It looks like this was released in v0.13.0 in 2019-03-11 so closing the issue. I'm about to make another release for Mirage 4
